MiniAn/minian_run.py

```diff
@@ -91,12 +91,12 @@
             params[params_.replace('params_','')+'-'+key] = value
 
 params_LocalCluster = dict(
-    n_workers=int(os.getenv("MINIAN_NWORKERS", 4)),
-    memory_limit="3GB",
-    resources={"MEM": 1},
-    threads_per_worker=2,
-    dashboard_address=":8787",
-    local_directory=dpath
+    n_workers = 4,
+    memory_limit = "auto",
```

Comment:
How does that work?

Reply:
"it fix partially some memory error" -- which part?

Reply:
@kapilseh please review as well

Reply:
The `memory_limit = "auto"` fix part of some memory problem. If the fixed value of 3Gb was not enough and it need like 3.2 or just a bit more now it's should work. Because by setting it with auto MiniAn will automatically set the memory limit depending on the user hardware.

Reply:
But it can potentially take over the whole RAM and crash, can't it?

Reply:
Shouldn't be the case. It didn't crash on my side. I think kapil tried it and it work well too.

On what I understand it will stop before overloading the memory. 
Because there is a limit it will stop. If it was set to no limit it will overload the memory.

Reply:
So to my question "how does that work", it will take as much memory as possible without overloading it.
But still don't get why it's "part" and why it's some "some" of memory issue -- I guess it's just a weird way of phrasing it..
Thanks!

Reply:
> So to my question "how does that work", it will take as much memory as possible without overloading it. 

Exaclty 

>But still don't get why it's "part" and why it's some "some" of memory issue -- I guess it's just a weird way of phrasing it.. Thanks!

And yes it is a weird way of phrasing it. Because at the end the memory problem came from the  memory_limit + saving . So when I did this pull request in my head I only fix part of the problem. :/